harper-core/src/linting/plural_conjugate.rs

```diff
@@ -101,14 +101,75 @@ mod tests {
         );
     }
 
-    // TODO disabled since it relied on the /1 affix flag which meant noun as well as singular
-    // TODO currently there is no way to mark number on pronouns without making them also nouns
-    // #[test]
-    // fn pronoun_singular() {
-    //     assert_suggestion_result(
-    //         "If he are testing it.",
-    //         PluralConjugate::default(),
-    //         "If he is testing it.",
-    //     );
-    // }
+    #[test]
```

Comment:
By the way, these tests are about what it termed `agreement` in grammar and linguistics. This means two (or more) words have a property in common, such as number, tense, case, person, etc. that must match. When two things don't match we don't necessarily know which of the two is wrong, the pronoun or the verb/copula. So a fully developed linter should suggest not only the verbs that match "he" but the pronouns that match "are". Since "he" is a third person singular subject pronoun we would suggest the third person plural subject pronoun "they". In contrived sentences it can seem very obvious which was intended but in real world sentences from non-native speakers it can go either way. Especially with "this" vs "these".